src/components/sidebarContent/visualization/SidebarVisualization.vue

```diff
@@ -100,38 +123,59 @@ export default {
     >
       <div class="buttons-container">
         <RadioCustom
-          v-for="sortingOption in mainSortingOptions"
+          v-for="sortingOption in secondarySortingOptions"
           :key="sortingOption.title"
           :label="sortingOption.title"
           :value="sortingOption.type"
           :model-value="sorting"
           name="sorting"
           @change="_setSorting(sortingOption.type)"
         />
-        <ButtonText
-          v-if="sorting === 'shuffled'"
-          class="randomize"
-          text="randomize!"
-          style-type="underline"
-          @click="toggleShuffleTrigger"
+      </div>
+      <div class="buttons-container paired">
+        <RadioCustom
+          :label="mainSortingOptions.custom.title"
+          :value="mainSortingOptions.custom.type"
+          :model-value="sorting"
+          name="sorting"
+          @change="_setSorting(mainSortingOptions.custom.type)"
         />
+        <div class="draggable-mode-section">
+          <ToggleSwitch
+            :is-checked="isDraggableMode"
+            title="reorder mode"
+            @change="_toggleDraggableMode"
+          />
+          <PopupBox
+            button-style-type="hint"
+            stop-propagation
+            position="lower-left"
+          >
+            <span>
+              Entering this mode, you will be able to sort your list manually by 
+              dragging and dropping items in the order you like. 
+              Note: all your filters and visualization will be reset.
+            </span>
+          </PopupBox>
```

Comment:
maybe put it in the component and reuse?
you have almost the same two peaces of code
here and in main screen